Doc/library/decimal.rst

```diff
@@ -1033,10 +1033,10 @@ New contexts can also be created using the :class:`Context` constructor
 described below. In addition, the module provides three pre-made contexts:
 
 
-.. class:: BasicContext
+.. data:: BasicContext
```

Comment:
Directives for `ExtendedContext` and `DefaultContext` must also be updated.

Reply:
I've updated the directives for ExtendedContext and DefaultContext and ensured the terminology matches the original intent. Please review again. 😊